### ISSUE TITLE
Pikachu ball saver crashes game

### ISSUE THREAD

**Issue by Denshadego** [huderlem/pokepinball-generations#9]
As the title says , when catching the ball with the pikachu ball saver (the one that goes left or right) the game crashes the emulator, possibly related to the "pika" sound it's supposed to do before the crash. Tested with gambatte emulator and it does not happen on the regular Pokemon pinball game (the sound plays and the game continues).

Thanks for the hack , its awesome. I hope this can be fixed.

**Comment by huderlem** [comment]
This sounds like an emulator-specific issue, since this crash doesn't happen on the TwitchPlaysPokemon stream or with the BGB emulator.  It's still worth looking into, though.  Just to confirm, the crash _does_ happen on gambatte, correct?  It's hard to tell from your message.  Also, did you build the ROM from the `master` branch of this repo, or did you download a patch from somewhere?  `master` might not be stable, and it's possible compiling with a newer version of RGBDS might result in the crash.

**Comment by Denshadego** [comment]
Yup, gambatte on retroarch. Used the ips posted on a forum, 1.0.

**Comment by huderlem** [comment]
Would you be able to point me to the patch?  Also, would you be able to test on the BGB emulator, too?  I’m worried your ROM might not quite be patched correctly, but it’s hard to say without seeing the patch file you used. Providing the ROM’s md5 or sha hash would help determine that, too.

**Comment by Denshadego** [comment]
Right, tried the same rom I have with bgb, and it does not happen there, so the rom it's fine I suppose. What's weird is that the regular Pokemon Pinball game does not have this problem on gambatte, but the patched one crashes.
This is the ips.
https://www.mediafire.com/file/9sdz35h7zhe2icd/PinballGenerations-1.0.ips/file
